fahrschein-http-api/src/main/java/org/zalando/fahrschein/http/api/RequestHandler.java

```diff
@@ -0,0 +1,10 @@
+package org.zalando.fahrschein.http.api;
+
+public interface RequestHandler {
+
+    void beforeExecute(Request request);
```

Comment:
```suggestion
    void beforeExecution(Request request);
```

Reply:
Maybe just `after`, `before` and `onError` should be enough.